noteorganiser/NoteOrganiser.py

```diff
@@ -128,6 +139,20 @@ def setExternalEditor(self):
             #Recover the field
             self.info.externalEditor = self.popup.commandline
 
+    def toggleRefresh(self):
+        """
+        toggle if the editor gets refreshed automatically when the file
+        changes
+        """
+        #save settings
+        self.info.refreshEditor = not self.info.refreshEditor
+        self.settings = QtCore.QSettings("audren", "NoteOrganiser")
+        self.settings.setValue("refreshEditor", self.info.refreshEditor)
+        if self.tabs.currentWidget == Editing(self):
```

Comment:
You are creating an instance of Editing just for comparison. It would be better to use `self.state`, however it is currently not very well implemented. But it would be better to fix this comparison (at least use a `isinstance`).